react/ui-components/src/molecules/stories/SuccessPanelCard.stories.js

```
@@ -0,0 +1,252 @@
+    variation={"primary"}
+    label="Button"
+    onClick={() => console.log("Clicked Button 2")}
+  />,
+];
+
+const children = [
+  <InfoCard
+    variant={"success"}
+    className={"panelcard-infocard"}
+    text={"This is success"}
+  />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)